tailscale/client.go

```diff
@@ -373,8 +373,15 @@ type (
 	}
 
 	NodeAttrGrant struct {
-		Target []string `json:"target,omitempty" hujson:"Target,omitempty"`
-		Attr   []string `json:"attr,omitempty" hujson:"Attr,omitempty"`
+		Target []string                       `json:"target,omitempty" hujson:"Target,omitempty"`
```

Comment:
I am mindful that this is a public PR. I believe there is other documentation which should be updated before we might proceed to implement this in the go-client.

Personally, the alternative you mentioned in Slack seems preferable: preserve the HuJSON.